clients/src/main/java/org/apache/kafka/common/ElectionType.java

```diff
@@ -18,10 +18,11 @@
 package org.apache.kafka.common;
 
 import java.util.Arrays;
+import java.util.Set;
 import org.apache.kafka.common.annotation.InterfaceStability;
```

Comment:
```suggestion
import org.apache.kafka.common.annotation.InterfaceStability;

import java.util.Arrays;
import java.util.Set;
```

Reply:
Is there a rule of imports layout? I observe the following layout in project.
```java
import org.apache.kafka.common.KafkaFuture;
import org.apache.kafka.common.Node;
import org.apache.kafka.common.acl.AclOperation;
import org.apache.kafka.common.annotation.InterfaceStability;

import java.util.Collection;
import java.util.Set;
```

```
import java.util.Set;
import java.util.concurrent.ConcurrentMap;
import java.util.concurrent.atomic.AtomicInteger;
import org.apache.kafka.clients.ApiVersions;
import org.apache.kafka.clients.producer.Callback;
import org.apache.kafka.common.utils.ProducerIdAndEpoch;
import org.apache.kafka.common.Cluster;
```